.github/ISSUE_TEMPLATE.md

```diff
@@ -1,3 +1,10 @@
+<!--
+Note that this is the issue tracker for OpenShift Origin.
+
+OpenShift Installer issues should be reported at https://github.com/openshift/openshift-ansible/issues
```

Comment:
maybe do similar for web-console? cc @jwforres

Reply:
Yes please. Console issues get filed in the wrong place a lot.

Reply:
@jwforres Added. PTAL.